src/pages/Dogs/Dogs.jsx

```diff
@@ -24,8 +24,19 @@ const Dogs = () => {
   const { backend } = useBackend();
   const [data, setData] = useState([]);
   const [facilityFilter, setFacilityFilter] = useState('');
+  const [filter, setFilter] = useState('');
   const [searchDog, setSearchDog] = useState('');
   const [dogs, setDogs] = useState([]);
+  const [checkedDogs, setCheckedDogs] = useState([]);
+
+  const getCheckedDogs = checkedDog => {
+    if (checkedDogs.includes(checkedDog)) {
+      setCheckedDogs(checkedDogs.filter(i => i != checkedDog));
```

Comment:
🚫 **[eslint]** <[eqeqeq](https://eslint.org/docs/rules/eqeqeq)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected '!==' and instead saw '!='.